bitnami/matomo/Chart.yaml

```
@@ -39,4 +39,4 @@ maintainers:
 name: matomo
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/matomo
-version: 5.4.1
+version: 5.4.2
```

There probably was a collision with versioning when integrating more recent changes from `main` branch to this one and the version is now outdated. I believe this should be a minor bump, as you are adding a new feature

```suggestion
version: 5.5.0
```